.github/workflows/ci_on_pr.yml

```
@@ -26,27 +26,28 @@
         include:
           - {os: ubuntu-22.04, compiler: spdx_license_lint}
           - {os: ubuntu-22.04, compiler: gcc13-cpp20-release}
-          - {os: ubuntu-22.04, compiler: gcc14-setup_only-lcov
-                                         gcc14-nosetup-cpp20-release
-                                         gcc14-nosetup-cpp23-release
-                                         gcc14-nosetup-cpp20-lcov
-                                         gcc14-nosetup-cpp20-debug-sanitize_address
-                                         gcc14-nosetup-cpp20-debug-sanitize_undefined
-                                         gcc14-nosetup-cpp20-debug-sanitize_thread
+          - {os: ubuntu-22.04, compiler: gcc14-cpp20-release}
+          - {os: ubuntu-25.04, compiler: gcc15-setup_only-lcov
```

actionlint (1.7.7)

label "ubuntu-25.04" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)